File: 18 Co wyświetli podany kod/treść.py
from math import isqrt, gcd, lcm

x = 67
x &= 6
x |= 7
x //= x ** 3
x += 3
x += gcd(16, 4)
x ^= lcm(8, 32)
x = isqrt(x)

if x == 6:
    print("Sean")
elif x == 7:
    print("Jeffrey")
elif x == 9:
    print("Marcin")
else:
    print("Stuart")

File: 18 Co wyświetli podany kod/treść.cpp
#include <iostream>
#include <numeric>
#include <cmath>

using namespace std;

int main()
{
    int x = 67;
    x &= 6;
    x |= 7;
    x /= (x * x * x);
    x += 3;
    x += gcd(16, 4);
    x ^= lcm(8, 32);
    x = sqrt(x);

    if (x == 6)
        cout << "Sean";
    else if (x == 7)
        cout << "Jeffrey";
    else if (x == 9)
        cout << "Marcin";
    else
        cout << "Stuart";
}
